configs/sample_params/src/alias.rs

```diff
@@ -0,0 +1,93 @@
+//! Aliases that resolve with reference to [`crate::SampleParams`].
+
+use std::str::FromStr;
+
+use iroha_data_model::prelude::{AccountId, AssetId};
+
+/// Alias of `T`. Can be parsed to `T: FromStr` when the string is provided by [`resolve`](Alias::resolve).
+pub trait Alias<T: FromStr>
+where
+    <T as FromStr>::Err: core::fmt::Debug,
+{
+    /// Parse [`self`] to `T`, assuming [`self`] should successfully [`resolve`](Alias::resolve).
+    ///
+    /// # Panics
+    ///
+    /// - [`resolve`](Alias::resolve) implementation is not compatible with [`FromStr`] for `T`
+    ///
+    /// # Example
+    ///
+    /// See [`crate`] documentation.
+    #[inline]
+    fn parse_alias(&self) -> T {
```

Comment:
Here I'd choose a short method name.
I think "parse_" implies this is an utility for tests because aliases would not be able to simply be parsed in a more authentic environment

Reply:
Well, maybe at least make an emphasis in the documentation that this crate and this function is not intended to be used anywhere apart from tests?